extensions/wikia/AdEngine/js/slot/service/slotRegistry.js

```diff
@@ -68,10 +70,20 @@ define('ext.wikia.adEngine.slot.service.slotRegistry',  [
 		slotQueueCount = {};
 	});
 
+	function storeScrollY(slotName) {
+		scrollYOnDfpRequest[slotName] = win.scrollY || win.pageYOffset;
```

Comment:
discussed offline, for ie < 9 we'd need to perform `document.documentElement.scrollLeft + document.documentElement.scrollTop` which works ONLY on those old ones IEs. We don't support IE at all: http://community.wikia.com/wiki/Help:Supported_Browsers